modules/ztp-troubleshooting-the-managed-cluster.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-troubleshooting-the-managed-cluster_{context}"]
+= Troubleshooting the managed cluster
+
```

Comment:
If you're styling this as a procedure that you mean for the user to complete as an E2E workflow, you need to include an introduction statement that tells them what they're troubleshooting.